pkg/render/monitor/monitor.go

```diff
@@ -57,6 +60,26 @@ const (
 	prometheusServiceAccountName = "prometheus"
 )
 
+var log = logf.Log.WithName("monitor_render")
+
+var operatorServiceAccountName = ""
+
+func init() {
```

Comment:
@tmjd Manually tested the environment variable `OPERATOR_SERVICEACCOUNT` in the cluster and the ServiceAccount name will change accordingly. I tried to add another `init()` function in UT to set the environment variable but this `init()` runs first. I need some suggestions on how to test this.